dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -871,8 +795,7 @@ function bindGameKeys(socket) {
     } else if (!player_item && item_at_pos && item_at_pos.portable) {
       // If there's a portable item here and we don't something in hand, pick it up.
       msg_type = "item_pick_up";
-      item_at_pos.position = null;
-      delete itemPositions[position[0] + '_' + position[1]];
+      gridItems.remove(position);
```

Comment:
`gridItems.remove()` seems to take an item, not a position, though perhaps this would be better.

Reply:
Looks like I passed an Item in one case and a position in another. I agree `position` is better.